Content.Shared/_RMC14/Xenonids/Construction/SharedXenoConstructionSystem.cs

```diff
@@ -100,6 +114,12 @@ public sealed class SharedXenoConstructionSystem : EntitySystem
     private static readonly ProtoId<TagPrototype> StructureTag = "Structure";
     private static readonly ProtoId<TagPrototype> PlatformTag = "Platform";
 
+    private static readonly string ResinDoorCorpseAddCollisionTag = "ResinDoorCorpseCollisionDelay";
```

Comment:
This doesn't look used so it should be removed, either way it should be typed ProtoId if you wanted to use it